packages/components/src/components/post-avatar-picture/post-avatar-picture.tsx

```diff
@@ -0,0 +1,85 @@
+import { Component, h, Host, Prop, State } from '@stencil/core';
+import { version } from '@root/package.json';
+import sha256 from 'crypto-js/sha256';
```

Comment:
This dependency might not be necessary, we could write our own function: 
https://remarkablemark.medium.com/how-to-generate-a-sha-256-hash-with-javascript-d3b2696382fd